### ISSUE TITLE
JGit runs in debug mode

### ISSUE THREAD

**Issue by selvakn** [TWChennai/gocd-git-path-material-plugin#6]
Looks like jgit runs in debug mode and it spits out a lot of debug log statements, which are forwarded to go-server.out.log.

There is another issue in the go server [https://github.com/gocd/gocd/issues/2811] and the log file go-server.out.log is not rotated. 

It results in the log file taking the entire disk space.

While the rotation issue shall be handled in the go server, there is no reason for the jgit and the plugin to run in debug mode.

**Comment by jskswamy** [comment]
@selvakn I've update the debug statement log level, check whether this reduces the logging else we need to explicitly set the logging level for JGit in go-server. refer [here](https://logging.apache.org/log4j/1.2/manual.html) to configure the logging level for JGit

**Comment by jskswamy** [comment]
@selvakn, there is no change in the go-server logging behavior after this commit, we need to turn the logging level for jGit in go-server to solve this

**Comment by selvakn** [comment]
Already tried it.. but for some reason, jgit doesnt seem to respect the log level in the log4j config of go-server :(

**Comment by jskswamy** [comment]
Temporarily I've fixed it with the help of truncate command and a 5 minutes cron job

```sh
crontab -e
```

and add the following content to it as go user

```sh
MAILTO="" #to disable cron job sending email once the job is completed
*/5 * * * * truncate -s 0 /var/log/go-server/go-server.out.log
```

> caveat: logs are lost, for debugging we can disable the cron job

**Comment by jskswamy** [comment]
Fixed [76c3a612](https://github.com/TWChennai/gocd-git-path-material-plugin/commit/76c3a61215f1fe1c87a66b6d9a844b4e5d926958)
